src/modules/coreHttpApi/controllers/IdentityDeletionController.ts

```diff
@@ -0,0 +1,24 @@
+import { TransportServices } from "@nmshd/runtime";
+import { Inject } from "@nmshd/typescript-ioc";
+import { Accept, Context, DELETE, Path, ServiceContext } from "@nmshd/typescript-rest";
+import { Envelope } from "../../../infrastructure";
+import { RouteNotFoundError } from "../../../infrastructure/httpServer/middlewares/genericErrorHandler";
+import { BaseController } from "../common/BaseController";
+
+@Path("/api/v2/IdentityDeletion")
+export class IdentityDeletionController extends BaseController {
+    public constructor(@Inject private readonly transportServices: TransportServices) {
+        super();
+    }
+
+    @DELETE
+    @Path("/")
+    @Accept("application/json")
+    public async getIdentityInfo(@Context context: ServiceContext): Promise<Envelope> {
+        if (!context.request._debugMode) {
```

Comment:
I'd rather like this controller to not be loaded at all instead of passing a debug mode flag that can easily get lost and then show this route to the public.

As said before, please remove all the routes and put them to another PR, we need to find better ways to provide debug apis - as this is not the way.